staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1275,3 +1281,51 @@ type RenameOptions struct {
 	NewName         string  `json:"newName"`
 	OldName         *string `json:"oldName,omitempty"`
 }
+
+// KubeVirtConfiguration holds all kubevirt configurations
+// +k8s:openapi-gen=true
+type KubeVirtConfiguration struct {
+	ResourceVersion             string                  `json:"resourceVersion,omitempty"`
```

Comment:
The ResourceVersion should be the one from the ConfigMap or the KubeVirtCR. Do we have a way to not have it included here?